services/app-api/handlers/reports/create.ts

```diff
@@ -135,14 +127,35 @@ export const createReport = handler(
       targetPopulations: "objectArray",
     };
 
+    const currentDate = Date.now();
+    let reportYear: number =
+      reportType === ReportType.WP
+        ? new Date(convertDateUtcToEt(currentDate)).getFullYear()
+        : workPlanMetadata!.reportYear;
+
+    let reportPeriod: number =
+      reportType === ReportType.WP
+        ? calculatePeriod(currentDate, workPlanMetadata)
+        : workPlanMetadata!.reportPeriod;
+
+    const overrideCopyOver =
+      unvalidatedMetadata?.copyReport &&
+      unvalidatedMetadata?.copyReport?.isCopyOverTest;
+
+    if (overrideCopyOver) {
+      reportYear = reportPeriod == 2 ? reportYear + 1 : reportYear;
+      reportPeriod = reportPeriod == 1 ? 2 : 1;
```

Comment:
I'm having trouble understanding line 147.  If it's an override copy over, and the report period is `1`, then it should be `2`, otherwise it'll be set to `1`?